src/test_oop_analysis.py

```diff
@@ -1,8 +1,28 @@
 import unittest
+from oop_analysis import in_out
 import pandas as pd
 import numpy as np
 from oop_analysis import analysis
 
+class test_in_out(unittest.TestCase):
+    def test_read_csv_as_df(self):
+
+        # given
+        file_in = r"J:\01_Projekte\SSC_Workshop\my-ssd-test-repo\data\expec.t"
```

Comment:
I'm wondering if it wouldn't be a better idea to write a csv file in the test, save it to file, then test the reading of the file in the test and delete it again at the end of the test

or at least use a **relative path** to a file that is in the GitHub repo :)